### ISSUE TITLE
Error unable to verify the first certificate

### ISSUE THREAD

**Issue by sys-maketecheasier** [microsoft/vscode-cpptools#605]
When trying to install the C/C++ extension, Visual Studio Code gives me the error:
Error unable to verify the first certificate
What should i do?

**Comment by sean-mcmanus** [comment]
What OS are you on? Does this repro for other extensions? What does it say in the Output window?

**Comment by zinechant** [comment]
@sean-mcmanus.  Hi. I am running Ubuntu 16.04 and ran into the same situation.
uname -a: Linux 4.4.0-78-generic #99-Ubuntu SMP Thu Apr 27 15:29:09 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
VSCode:1.12.2
![image](https://cloud.githubusercontent.com/assets/7436105/26433392/0e5155ea-40c8-11e7-9cbe-7a2c0fccc38f.png)

**Comment by rlax** [comment]
@sys-maketecheasier @sean-mcmanus 
I guess this issue will help.
https://github.com/Microsoft/vscode/issues/155

**Comment by bobbrow** [comment]
TL;DR -  try setting `"http.proxyStrictSSL": false` in your settings.json
